motoko/ios-notifications/dapp-demo/src/www/src/pages/about.ts

```diff
@@ -0,0 +1,17 @@
+import { html, render } from "lit-html";
+import { Page, PageOptions } from "../types";
+
+const content = (homeUrl: string) => html`<div class="container">
```

Comment:
Does it make sense to have this page at all? I would either remove it or make it more useful (by linking back to the example in the github repo for example).

Reply:
Ah, I just realized this is used in the push notification show-case. Still, linking back to the examples repo would be nice. 😉